plugins/catalog-backend-module-github/api-report.md

```diff
@@ -131,6 +148,12 @@ export interface GitHubOrgEntityProviderOptions {
   logger: Logger;
   orgUrl: string;
   schedule?: 'manual' | TaskRunner;
+  teamTransformer?: TeamTransformer;
+  // (undocumented)
+  teamTransformerOptions?: TeamTransformerOptions;
```

Comment:
Namespace is still used in the default transformers however and is passed in from some of the other functionality. Such as in `GithubMultiOrgReaderProcessor`.

So i have added it to the parameters that are passed through rather than implementing another default transformer or some kind of wrapper.